languageFactories/wdl-draft2/src/main/scala/languages/wdl/draft2/WdlDraft2LanguageFactory.scala

```diff
@@ -1,32 +1,36 @@
 package languages.wdl.draft2
 
+import java.security.MessageDigest
+import java.util.concurrent.{Callable, TimeUnit}
+
+import cats.data.EitherT.fromEither
+import cats.effect.IO
 import cats.instances.either._
 import cats.instances.list._
 import cats.syntax.functor._
-import cats.data.EitherT.fromEither
-import cats.effect.IO
 import cats.syntax.traverse._
+import com.google.common.cache.{Cache, CacheBuilder}
 import common.Checked
-import common.validation.Validation._
 import common.validation.Checked._
 import common.validation.ErrorOr.{ErrorOr, _}
 import common.validation.Parse.Parse
+import common.validation.Validation._
 import cromwell.core._
 import cromwell.languages.util.ImportResolver.{ImportResolutionRequest, ImportResolver}
 import cromwell.languages.util.{ImportResolver, LanguageFactoryUtil}
 import cromwell.languages.{LanguageFactory, ValidatedWomNamespace}
+import languages.wdl.draft2.WdlDraft2LanguageFactory._
 import wdl.draft2.model.{Draft2ImportResolver, WdlNamespace, WdlNamespaceWithWorkflow}
 import wdl.shared.transforms.wdlom2wom.WdlSharedInputParsing
 import wdl.transforms.draft2.wdlom2wom.WdlDraft2WomBundleMakers._
-import wom.core.{WorkflowJson, WorkflowOptionsJson, WorkflowSource}
-import wom.graph.GraphNodePort.OutputPort
 import wdl.transforms.draft2.wdlom2wom.WdlDraft2WomExecutableMakers._
+import wom.core.{WorkflowJson, WorkflowOptionsJson, WorkflowSource}
 import wom.executable.WomBundle
 import wom.expression.IoFunctionSet
-import wom.transforms.WomExecutableMaker.ops._
+import wom.graph.GraphNodePort.OutputPort
 import wom.transforms.WomBundleMaker.ops._
-import wom.values.WomValue
-import languages.wdl.draft2.WdlDraft2LanguageFactory._
+import wom.transforms.WomExecutableMaker.ops._
+import wom.values.{WomValue, _}
```

Comment:
This import, and others imports this file, have been Intellij-ed and should be fixed.